echo/server/dembrane/tasks.py

```
@@ -106,6 +107,14 @@ def init_sentry_celery(**_kwargs):
     init_sentry()
 
 
+@celery_app.on_after_configure.connect
+def setup_periodic_tasks(sender: Celery, **_kwargs):
+    sender.add_periodic_task(
+        crontab(hour=7, minute=30, day_of_week=1),
+        test_collect_unfinished_conversations.s(),
```

Ruff (0.8.2)

Undefined name `test_collect_unfinished_conversations`

(F821)